index.html

```diff
@@ -1,8 +1,153 @@
 <!DOCTYPE html>
 <html>
    <head>
+      <link rel="stylesheet" type="text/css" href="styles.css">
+      <title>Animal Behavior</title>
    </head>
+
    <body>
-      <!-- Write your HTML in this file! -->
+      <header>
+         <h3 id = "mainHeading">To <s>All</s> Some of the Pets I've Loved Before</h3>
+     <h4>Cats</h4>
+       <nav><ul><li><a href = #callieholly>Holly &amp; Callie</a> |</li>
+         <li><a href = #nugget>Nugget</a> | </li>
+         <li><a href = #accio>Accio</a> | </li>
+         <li><a href = #kahlua>Kahlua</a> | </li>
+         <li><a href = #pebbles>Pebbles</a></li>
+         </nav></ul>
+
+      <h4>Dogs</h4>
+         <nav><ul>
+         <li><a href = #bud>Bud</a> | </li>
+         <li><a href = #malibu>Malibu</a></li>
+         </ul></nav>
+         </header>
+
+      
+      <main>
+         
+      <h1>Cats</h1>
+         
+      <article id = "callieholly">
+      <h2>Callie & Holly</h2>
+      <p>Callie came to my family as a stray when she was about a year old. She was the first animal I really have core memories with. She was an avid mouser and loved exploring the outdoors. Callie remained a well-fed outdoor cat until, one day, she seemingly dropped all of her weight. I was five at the time and didn't really notice, but my parents did. A few weeks later, while doing yardwork, my mother noticed a tiny black and white face peering at her from a small hole in the door of a shed on our property. Behold, Callie had given birth to six kittens of varying colors and patterns. Out of this litter, we kept the runt, the lone other calico cat. I named her Holly because I loved the name and had heard my sister talk about one of her friends named Holly. (Unfortunatly, human Holly would later pass away. I befriended her daughter as adults and shared with her the story of how I inadvertently named a cat after her late mom.) Both cats have since crossed the rainbow bridge but they started my love for cats in general but especially calicos.</p>
```

Comment:
When it comes to paragraphs like this, for readability, I recommend hitting enter after every sentence or so, so you don't have such a long single line where you need to scroll over so much. Don't worry, this won't change how it appears on your webpage, but just makes the code a little more readable!